activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+*   Add option to merge association scopes to subqueries.
+
+    It removes a breaking change. Conditions that were not applied in
+    previous versions would be applied now giving people different results.
```

Comment:
The changelog is relative to the previous release, so this entry should be folded into the previous one